### DISCUSSION TITLE
no access to copilot

### DISCUSSION THREAD

**Discussion by ghost** [community/community#106887] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hi Team , 

  I have been granted access to github copilot license by my organization . But I am unable to login to the copilot extension using the credentials. Where can I get support on this ?

Thanks
Sabya

**Comment by davecheney** [comment, marked as answer]
@sabyagit there is no copilot subscription associated with this Github account. If your employer has invited you to join a copilot enabled organisation you must first accept that invitation and complete whatever steps are required -- for example, the organisation may require that 2fa be configured for your account -- for the organisations copilot subscription to take effect. 

Conversely, if the organisation has chosen to assign seats directly to individual members of the organisation, they will have to complete that step.

In either case you will receive notification via email to your accounts primary email address.
